app/src/main/res/values/strings.xml

```diff
@@ -69,7 +69,7 @@
     <string name="be_sure_to_make_a_backup">Be sure to make a backup soon!</string>
     <string name="when_you_press_the_done">When you press the "Done" button, we\'ll take you to the same screen you\'ll see after scanning your DiceKey from the home screen.</string>
 
-    <string name="please_connect_a_solokey">Please connect a SoloKey.\n\nMake sure you have enabled OTG option in Android Settings.</string>
+    <string name="please_connect_a_solokey">Please connect a SoloKey.\n\nMake sure you have enabled the OTG option in Android Settings.</string>
```

Comment:
Just remove `Make sure you have enabled the OTG option in Android Settings` as we should not tell a user to do something using a acronym they don't know.